frontend/elements/src/module/_groups/cards/play/card/card.ts

```diff
@@ -186,18 +190,18 @@ export class _ extends LitElement {
                 </div>
                 <div class="back">
                     ${doubleSided
-                        ? html`<div
+                ? html`<div
                               class="content"
                               style=${getContentStyle(
-                                  styleKind,
-                                  theme,
-                                  mode,
-                                  backSide
-                              )}
+                    styleKind,
+                    theme,
+                    mode,
+                    backSide
+                )}
                           >
                               <slot name="backSideContent"></slot>
                           </div>`
-                        : html`<img-ui
+                : html`<img-ui
```

Comment:
nitpick: check formatting. 

We unfortunately don't have prettier set up yet to auto-correct these.